go.mod

```diff
@@ -3,155 +3,80 @@ module github.com/kyma-project/template-operator
 go 1.19
 
 require (
+	github.com/ghodss/yaml v1.0.0
 	github.com/go-logr/logr v1.2.3
-	github.com/kyma-project/module-manager v0.0.0-20221125165158-b84c96f2a9db
 	github.com/onsi/ginkgo/v2 v2.5.0
 	github.com/onsi/gomega v1.24.1
 	golang.org/x/time v0.0.0-20220722155302-e5dcc9cfc0b9
-	k8s.io/api v0.25.2
-	k8s.io/apimachinery v0.25.2
-	k8s.io/client-go v0.25.2
+	k8s.io/api v0.25.4
```

Comment:
0.26.0 should already be out I believe

Reply:
0.26.0 had a a lot of incompatibility issues so I refrained from upgrading as part of this PR